pkg/api/types.go

```diff
@@ -2049,6 +2063,23 @@ type ServiceProxyOptions struct {
 	Path string
 }
 
+// OwnerReference contains enough information to let you identify an owning
+// object. Currently, an owning object must be in the same namespace, so there
+// is no namespace field.
+type OwnerReference struct {
+	// API version of the referent.
+	APIVersion string `json:"apiVersion,omitempty"`
```

Comment:
The garbage collector needs to understand which objects are mounted multiple places. It shouldn't matter which version of an object you put here.

It's more complex than just version-- consider that Job is mounted in batch/v1, batch/v2alpha1, and **extensions/v1beta1**.